### ISSUE TITLE
remove mappings to Quote templates

### ISSUE THREAD

**Issue by jimkont** [dbpedia/mappings-tracker#73]
These mappings produce more noise than actual facts, usually co-exist with other main templates and generate new intemediate nodes.

If people want them we should look at creating a special purpose extractor for them

**Comment by VladimirAlexiev** [comment]
Is this related to #66?

**Comment by jimkont** [comment]
yes, also to some of https://github.com/dbpedia/extraction-framework/issues/430 https://github.com/dbpedia/extraction-framework/issues/432,  https://github.com/dbpedia/extraction-framework/issues/434
